evalai/utils/text_styling.py

```diff
@@ -0,0 +1,27 @@
+
+from click import echo, style
+
+
+def print_message(message, error=False):
+
+    style_message = ""
+
+    index = 0
+
+    for i in list(message.split("`")):
+
+        if index % 2:
+            style_message += style(i, fg="cyan")
```

Comment:
Nice logic here. A small query here: what do you think about adding a check to see if the text is actually *enclosed* in backticks. That is, to check if there is a closing backtick and only applying this after that. 
I'm not sure if it is easy to do here, but if the parts of the messages were in a list `ls`:
```
if len(ls) == ODD:
  # apply command style
```

Let me know what you think.

Another small query: I think we can add `bold=True` here?

Reply:
Ya sure. both of them are doable and looks nice. I will follow it in the following PR

Reply:
Yes I will do that in following PR